src/platform/forms/save-in-progress/RoutedSavableApp.jsx

```diff
@@ -277,10 +278,12 @@ const mapDispatchToProps = {
 };
 
 export default withRouter(
-  connect(
-    getSaveInProgressState,
-    mapDispatchToProps,
-  )(RoutedSavableApp),
+  withToggleProvider(
```

Comment:
going to try to test this today. Might remove it before merging, though since it impacts every form.